lib/Mojolicious/Renderer.pm

```diff
@@ -108,6 +108,10 @@ sub render {
   # Inheritance
   my $content = $stash->{'mojo.content'} //= {};
   local $content->{content} = $output =~ /\S/ ? $output : undef if $stash->{extends} || $stash->{layout};
+  if ($stash->{inline_layout}) {
+    @$options{inline} = delete $stash->{inline_layout};
```

Comment:
`@$options` doesn't make sense here.

Reply:
I can't quite say what yet, but something about the logic seems wrong. 🤔

Reply:
Maybe that `inline` value needs to be `local`-ised to avoid side effects?

Reply:
You’re right that it should be possible to execute another layout defined in the inline layout (if we started from a non-inline template).

I localized the inline value and fixed another bug (a layout and inline_layout defined initially could have both been executed), and I observed that it prevented entering the second loop. However, I was not able to get it to work in the correct case (template => inline_layout => other layout defined in the inline_layout), so I was not able either to write a good test case for the fixed part.